### ISSUE TITLE
XML Marshalling is locale dependent

### ISSUE THREAD

**Issue by kaspernielsen** [gla-rad/S-100#1]
I tried running the test on my machine with a locale that has ',' as the decimal separator instead of '.' and the S100ExchangeSetUtilsTest.testMarchallS125 test fails because we get

```
Expected:
<posList>-10.0000000 -10.0000000</posList>
<posList>-11.0000000 10.0000000</posList>

Actual:
<posList>-10,0000000 -10,0000000</posList>
<posList>-11,0000000 10,0000000</posList>
```

Can be reproduced by anyone by temporarily changing this code.

    @Test
    void testMarchallS125() throws JAXBException {
        Locale.setDefault(Locale.FRANCE);  //Warning sets it globally, but good enough to reproduce

I think we only want to generate xml that always uses . for decimal separator?

**Comment by nvasta** [comment]
Good question... I don't think the positions number format in S-100 are based on the Locale.

I can raise a question though... for now let's assume it will always be a ".".

In the meatime, I will see if we can set the Marshaller Locale manually to not be system dependent.

**Comment by kaspernielsen** [comment]
Just found this in the S-124 Spec, I assume it is the same for every S-100 product

9.2 Encoding of Latitude and Longitude
* Values of latitude and longitude can be accurate up to 7 decimal places. Coordinates must be encoded as decimals in the format described below.

* Values should be coded as decimal numbers with 7 or fewer digits after the decimal. The normative encoding is in degrees, with an accuracy of 10-7 degrees, i.e., up to 7 digits after the decimal point.
* The decimal point must be indicated by the “.” character (punctuation).

* Trailing zeroes after the decimal point (and the decimal point itself if appropriate) may be omitted at producer discretion.

**Comment by nvasta** [comment]
That is correct, I was aware of the 7 decimal places. We can check S-100 for the decimal point a well.
